[E:runtime][1970-06-26 15:31:35 @lualib/bootstrap.lua:118] thread:6568 start,command:event@server/logger_server@101,lua mem:148.872070kb,c mem:194.914062kb
[E:runtime][1970-06-26 15:31:35 @./lualib/import.lua:24] handler.logger_handler local var:event:6,_log_ctx:8,_M:16,
[E:runtime][1970-06-26 15:31:35 @./lualib/import.lua:24] server.protocol_forward local var:
[E:runtime][1970-06-26 15:31:35 @./lualib/import.lua:24] server.rpc local var:connect_channel:18,do_send:61,CONNECT_STATUS:10,event:2,channel:5,
[E:runtime][1970-06-26 15:31:35 @./lualib/import.lua:24] handler.agent_handler local var:protocol:4,
[E:runtime][1970-06-26 15:31:35 @lualib/bootstrap.lua:118] thread:6569 start,command:event@server/agent@101,lua mem:202.244141kb,c mem:295.554688kb
[E:runtime][1970-06-26 15:31:44 @./lualib/import.lua:24] handler.agent_handler local var:protocol:4,
[E:runtime][1970-06-26 15:31:44 @lualib/bootstrap.lua:103] thread:21005 start,command:event@server/agent@101,lua mem:202.840820kb,c mem:296.468750kb
[E:runtime][1970-06-26 15:31:45 @./lualib/import.lua:24] handler.agent_handler local var:protocol:4,
[E:runtime][1970-06-26 15:31:45 @lualib/bootstrap.lua:103] thread:21022 start,command:event@server/agent@101,lua mem:202.840820kb,c mem:296.468750kb
[E:runtime][1970-06-26 15:31:45 @./lualib/import.lua:24] handler.agent_handler local var:protocol:4,
[E:runtime][1970-06-26 15:31:45 @lualib/bootstrap.lua:103] thread:21039 start,command:event@server/agent@101,lua mem:203.309570kb,c mem:296.984375kb
[E:runtime][1970-06-26 15:31:45 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:45 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:45 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:46 @./lualib/import.lua:24] handler.agent_handler local var:protocol:4,
[E:runtime][1970-06-26 15:31:46 @lualib/bootstrap.lua:103] thread:21057 start,command:event@server/agent@101,lua mem:203.421875kb,c mem:297.171875kb
[E:runtime][1970-06-26 15:31:46 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:46 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:46 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:46 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:46 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:46 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:46 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:47 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:47 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:47 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:48 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:48 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:48 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:48 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:49 @./lualib/import.lua:24] handler.agent_handler local var:protocol:4,
[E:runtime][1970-06-26 15:31:49 @lualib/bootstrap.lua:103] thread:21216 start,command:event@server/agent@101,lua mem:203.994141kb,c mem:298.015625kb
[E:runtime][1970-06-26 15:31:49 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:50 @./lualib/import.lua:24] handler.agent_handler local var:protocol:4,
[E:runtime][1970-06-26 15:31:50 @lualib/bootstrap.lua:103] thread:21236 start,command:event@server/agent@101,lua mem:203.998047kb,c mem:298.015625kb
[E:runtime][1970-06-26 15:31:50 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:50 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:51 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:51 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:51 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:51 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:51 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:51 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:51 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:51 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:52 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:52 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:52 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:52 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:52 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:52 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:52 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:52 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:52 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:52 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:52 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:31:52 @./script/server/agent.lua:56] error
[E:runtime][1970-06-26 15:33:31 @./lualib/import.lua:24] handler.agent_handler local var:protocol:4,
[E:runtime][1970-06-26 15:33:31 @lualib/bootstrap.lua:103] thread:31328 start,command:event@server/agent@101,lua mem:203.416016kb,c mem:297.156250kb
[E:runtime][1970-06-26 16:00:41 @./lualib/event.lua:71] ./lualib/event.lua:92: attempt to index a nil value (local 'addr')
stack traceback:
	./lualib/event.lua:92: in upvalue 'resolve_addr'
	./lualib/event.lua:124: in function 'event.connect'
	./script/server/scene.lua:70: in upvalue 'func'
	./lualib/event.lua:49: in function <./lualib/event.lua:48>
[E:runtime][1970-06-26 16:00:41 @lualib/bootstrap.lua:103] thread:24245 start,command:event@server/scene@101,lua mem:200.612305kb,c mem:431.289062kb
[E:runtime][1970-06-26 16:00:41 @./lualib/event.lua:71] ./lualib/event.lua:92: attempt to index a nil value (local 'addr')
stack traceback:
	./lualib/event.lua:92: in upvalue 'resolve_addr'
	./lualib/event.lua:124: in function 'event.connect'
	./script/server/scene.lua:34: in upvalue 'connect_server'
	./script/server/scene.lua:78: in upvalue 'func'
	./lualib/event.lua:49: in function <./lualib/event.lua:48>
[E:runtime][1970-06-26 16:00:41 @lualib/bootstrap.lua:103] thread:24272 start,command:event@server/scene@101,lua mem:200.050781kb,c mem:439.406250kb
[E:runtime][1970-06-26 16:00:41 @./lualib/event.lua:71] ./lualib/event.lua:92: attempt to index a nil value (local 'addr')
stack traceback:
	./lualib/event.lua:92: in upvalue 'resolve_addr'
	./lualib/event.lua:110: in function 'event.listen'
	./script/server/scene.lua:82: in upvalue 'func'
	./lualib/event.lua:49: in function <./lualib/event.lua:48>
[E:runtime][1970-06-26 16:00:41 @lualib/bootstrap.lua:103] thread:24276 start,command:event@server/scene@101,lua mem:199.957031kb,c mem:439.343750kb
[E:runtime][1970-06-26 16:00:41 @./lualib/event.lua:71] ./script/server/scene.lua:93: attempt to index a nil value (local 'master_channel')
stack traceback:
	./script/server/scene.lua:93: in upvalue 'func'
	./lualib/event.lua:49: in function <./lualib/event.lua:48>
[E:runtime][1970-06-26 16:00:41 @lualib/bootstrap.lua:103] thread:24291 start,command:event@server/scene@101,lua mem:200.457031kb,c mem:437.445312kb
[E:runtime][1970-06-26 16:00:42 @./lualib/event.lua:71] ./script/server/scene.lua:93: attempt to index a nil value (local 'master_channel')
stack traceback:
	./script/server/scene.lua:93: in upvalue 'func'
	./lualib/event.lua:49: in function <./lualib/event.lua:48>
[E:runtime][1970-06-26 16:00:42 @lualib/bootstrap.lua:103] thread:24317 start,command:event@server/scene@101,lua mem:200.457031kb,c mem:437.742188kb
[E:runtime][1970-06-26 16:00:42 @lualib/bootstrap.lua:103] thread:24321 start,command:event@server/scene@101,lua mem:200.850586kb,c mem:430.320312kb
[E:runtime][1970-06-26 16:00:42 @lualib/bootstrap.lua:103] thread:24324 start,command:event@server/scene@101,lua mem:200.855469kb,c mem:430.320312kb
[E:runtime][1970-06-26 16:00:45 @lualib/bootstrap.lua:103] thread:24493 start,command:event@server/scene@101,lua mem:200.850586kb,c mem:430.507812kb
[E:runtime][1970-06-26 16:00:45 @lualib/bootstrap.lua:103] thread:24516 start,command:event@server/scene@101,lua mem:200.850586kb,c mem:430.507812kb
[E:runtime][1970-06-26 16:00:46 @./lualib/event.lua:71] ./lualib/event.lua:92: attempt to index a nil value (local 'addr')
stack traceback:
	./lualib/event.lua:92: in upvalue 'resolve_addr'
	./lualib/event.lua:124: in function 'event.connect'
	./script/server/scene.lua:70: in upvalue 'func'
	./lualib/event.lua:49: in function <./lualib/event.lua:48>
[E:runtime][1970-06-26 16:00:46 @lualib/bootstrap.lua:103] thread:24541 start,command:event@server/scene@101,lua mem:200.651367kb,c mem:431.132812kb
[E:runtime][1970-06-26 16:00:48 @./lualib/event.lua:71] ./lualib/event.lua:92: attempt to index a nil value (local 'addr')
stack traceback:
	./lualib/event.lua:92: in upvalue 'resolve_addr'
	./lualib/event.lua:124: in function 'event.connect'
	./script/server/world.lua:29: in upvalue 'connect_server'
	./script/server/world.lua:65: in upvalue 'func'
	./lualib/event.lua:49: in function <./lualib/event.lua:48>
[E:runtime][1970-06-26 16:00:48 @lualib/bootstrap.lua:103] thread:24820 start,command:event@server/world@101,lua mem:204.273438kb,c mem:274.492188kb
[E:runtime][1970-06-26 16:01:01 @lualib/bootstrap.lua:103] thread:7244 start,command:event@server/master@101,lua mem:194.525391kb,c mem:262.132812kb
[E:runtime][1970-06-26 16:01:02 @lualib/bootstrap.lua:103] thread:7248 start,command:event@server/world@101,lua mem:197.192383kb,c mem:265.164062kb
[E:runtime][1970-06-26 16:01:02 @lualib/bootstrap.lua:103] thread:7275 start,command:event@server/scene@101,lua mem:198.307617kb,c mem:266.539062kb
[E:runtime][1970-06-26 16:01:02 @lualib/bootstrap.lua:103] thread:7293 start,command:event@server/agent@101,lua mem:205.070312kb,c mem:299.164062kb
[E:runtime][1970-06-26 16:02:53 @./lualib/event.lua:71] user.server.startup:34: attempt to index a nil value (global 'mongodb')
stack traceback:
	user.server.startup:34: in field 'run'
	./script/server/world.lua:31: in upvalue 'func'
	./lualib/event.lua:49: in function <./lualib/event.lua:48>
[E:runtime][1970-06-26 16:02:55 @./script/server/agent.lua:58] agent start success
[E:runtime][1970-06-26 16:10:17 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][1970-06-26 16:10:17 @./script/server/agent.lua:58] agent start success
[E:runtime][1970-06-26 16:10:18 @./script/server/agent.lua:58] agent start success
[E:runtime][1970-06-26 16:10:18 @./script/server/agent.lua:58] agent start success
[E:runtime][1970-06-26 16:10:20 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:14:25 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:17:21 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:25:48 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:25:49 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:26:37 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:26:38 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:27:21 @user.server.startup:49] connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:27:21 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:27:22 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:27:40 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:27:40 @user.server.startup:49] connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:27:41 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:28:21 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:28:21 @user.server.startup:49] connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:28:22 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:28:22 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:29:55 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:29:55 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:29:56 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:34:40 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:34:40 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:34:40 @user.server.startup:49] connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:34:41 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:34:41 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:40:52 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:40:52 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:40:53 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:40:53 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:41:48 @user.server.startup:49] connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:41:48 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:41:48 @user.server.startup:49] connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:41:48 @user.server.startup:49] connect server:login ipc://login.ipc failed:No such file or directory
[E:runtime][2018-04-13 09:41:49 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:41:49 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:43:26 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:43:26 @user.server.startup:49] connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:43:26 @user.server.startup:49] connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:43:26 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:43:27 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:43:27 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:44:27 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:44:27 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:44:28 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:44:28 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:45:19 @user.server.startup:49] connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:45:19 @user.server.startup:49] connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:45:19 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:45:49 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:45:49 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:45:49 @user.server.startup:49] connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:45:50 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:45:50 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:45:51 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:45:51 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:45:52 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:45:52 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:45:53 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:45:53 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:45:54 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:45:54 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:45:56 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:45:56 @user.server.startup:49] connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:45:56 @user.server.startup:49] connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:45:56 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:45:57 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:45:57 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:45:58 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:45:58 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:45:59 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:45:59 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:00 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:00 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:01 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:01 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:02 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:02 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:03 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:03 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:04 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:04 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:05 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:15 @user.server.startup:49] connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:15 @user.server.startup:49] connect server:master ipc://master.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:15 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:15 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:16 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:16 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:17 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:17 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:18 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:18 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:19 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:19 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:20 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:20 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:21 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:21 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:22 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:22 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:23 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:23 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:24 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:43 @user.server.startup:49] connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:43 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:43 @user.server.startup:49] connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:43 @user.server.startup:49] connect server:login ipc://login.ipc failed:No such file or directory
[E:runtime][2018-04-13 09:46:44 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:44 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:45 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:45 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:46 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:46 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:47 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:47 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:55 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:55 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:55 @user.server.startup:49] connect server:master ipc://master.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:55 @user.server.startup:49] connect server:master ipc://master.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:56 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:56 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:57 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:57 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:58 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:58 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:59 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:46:59 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:47:00 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:47:00 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:47:01 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:47:01 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:47:02 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:47:02 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:47:03 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:47:03 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:47:04 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:47:16 @user.server.startup:49] connect server:master ipc://master.ipc failed:Connection refused
[E:runtime][2018-04-13 09:47:16 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:47:16 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:47:16 @user.server.startup:49] connect server:master ipc://master.ipc failed:Connection refused
[E:runtime][2018-04-13 09:47:17 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:47:18 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:47:19 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:47:20 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:47:21 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:47:22 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:47:23 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:47:24 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:47:25 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:04 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:04 @user.server.startup:49] connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:04 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:05 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:05 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:06 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:06 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:07 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:07 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:08 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:08 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:09 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:09 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:10 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:10 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:11 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:11 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:12 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:12 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:13 @user.server.startup:49] connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:46 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:46 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:No such file or directory
[E:runtime][2018-04-13 09:48:46 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:47 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:47 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:48 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:48 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:49 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:49 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:50 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:50 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:51 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:51 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:52 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:48:52 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:49:29 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:49:29 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:49:29 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:49:30 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:49:30 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:49:31 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:49:31 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:49:32 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:49:32 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:49:33 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:49:33 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:49:34 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:49:34 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:49:35 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:49:35 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:49:36 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:49:36 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:49:37 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:49:37 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:49:38 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:50:09 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:50:09 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:50:10 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:50:11 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:50:12 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:50:13 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:50:14 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:50:15 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:50:16 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:50:17 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:50:18 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:51:08 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:51:08 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:51:08 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:51:08 @./script/server/login.lua:58] login listen client success
[E:runtime][2018-04-13 09:51:09 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:51:09 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:51:40 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:51:40 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:51:40 @./script/server/login.lua:58] login listen client success
[E:runtime][2018-04-13 09:51:41 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:51:41 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:52:12 @user.server.startup:49] server/scene connect server:master ipc://master.ipc failed:Connection refused
[E:runtime][2018-04-13 09:52:12 @user.server.startup:49] server/agent connect server:master ipc://master.ipc failed:Connection refused
[E:runtime][2018-04-13 09:52:12 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:52:12 @./script/server/login.lua:58] login listen client success
[E:runtime][2018-04-13 09:52:13 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:52:13 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:52:18 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:52:18 @user.server.startup:49] server/agent connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:52:18 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:52:18 @./script/server/login.lua:58] login listen client success
[E:runtime][2018-04-13 09:52:19 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:52:19 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:52:36 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:52:36 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:52:36 @./script/server/login.lua:58] login listen client success
[E:runtime][2018-04-13 09:52:36 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:No such file or directory
[E:runtime][2018-04-13 09:52:37 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:52:37 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:53:04 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:53:04 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:53:04 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:53:04 @./script/server/login.lua:58] login listen client success
[E:runtime][2018-04-13 09:53:05 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:53:05 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 09:53:27 @user.server.startup:49] server/scene connect server:master ipc://master.ipc failed:Connection refused
[E:runtime][2018-04-13 09:53:27 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:53:27 @./script/server/login.lua:58] login listen client success
[E:runtime][2018-04-13 09:53:27 @user.server.startup:49] server/agent connect server:master ipc://master.ipc failed:Connection refused
[E:runtime][2018-04-13 09:53:28 @./script/server/agent.lua:59] agent start success
[E:runtime][2018-04-13 09:54:09 @./script/server/agent.lua:59] agent start success
[E:runtime][2018-04-13 09:54:09 @user.server.startup:49] server/agent connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:54:09 @./script/server/login.lua:58] login listen client success
[E:runtime][2018-04-13 09:54:23 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:54:23 @user.server.startup:49] server/scene connect server:master ipc://master.ipc failed:Connection refused
[E:runtime][2018-04-13 09:54:23 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:54:23 @./script/server/login.lua:58] login listen client success
[E:runtime][2018-04-13 09:54:24 @./script/server/agent.lua:59] agent start success
[E:runtime][2018-04-13 09:54:36 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:54:36 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:54:36 @./script/server/login.lua:58] login listen client success
[E:runtime][2018-04-13 09:54:36 @user.server.startup:49] server/agent connect server:master ipc://master.ipc failed:Connection refused
[E:runtime][2018-04-13 09:54:37 @./script/server/agent.lua:59] agent start success
[E:runtime][2018-04-13 09:54:55 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:54:55 @./script/server/login.lua:58] login listen client success
[E:runtime][2018-04-13 09:54:56 @./script/server/agent.lua:59] agent start success
[E:runtime][2018-04-13 09:56:00 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:56:00 @./script/server/login.lua:58] login listen client success
[E:runtime][2018-04-13 09:56:01 @./script/server/agent.lua:59] agent start success
[E:runtime][2018-04-13 09:56:24 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:56:24 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:56:24 @./script/server/login.lua:58] login listen client success
[E:runtime][2018-04-13 09:56:25 @./script/server/agent.lua:59] agent start success
[E:runtime][2018-04-13 09:58:02 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:58:02 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:58:02 @./script/server/login.lua:58] login listen client success
[E:runtime][2018-04-13 09:58:03 @./script/server/agent.lua:59] agent start success
[E:runtime][2018-04-13 09:59:16 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 09:59:16 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:59:16 @./script/server/login.lua:58] login listen client success
[E:runtime][2018-04-13 09:59:17 @./script/server/agent.lua:59] agent start success
[E:runtime][2018-04-13 09:59:50 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 09:59:50 @./script/server/login.lua:58] login listen client success
[E:runtime][2018-04-13 09:59:51 @./script/server/agent.lua:59] agent start success
[E:runtime][2018-04-13 10:00:22 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 10:00:22 @./script/server/login.lua:58] login listen client success
[E:runtime][2018-04-13 10:00:22 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 10:00:23 @./script/server/agent.lua:59] agent start success
[E:runtime][2018-04-13 10:01:02 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 10:01:02 @./script/server/login.lua:58] login listen client success
[E:runtime][2018-04-13 10:01:03 @./script/server/agent.lua:59] agent start success
[E:runtime][2018-04-13 10:02:58 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 10:02:58 @./script/server/login.lua:58] login listen client success
[E:runtime][2018-04-13 10:02:59 @./script/server/agent.lua:59] agent start success
[E:runtime][2018-04-13 10:03:31 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 10:03:31 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 10:03:31 @./script/server/login.lua:59] login listen client success
[E:runtime][2018-04-13 10:03:32 @./script/server/agent.lua:59] agent start success
[E:runtime][2018-04-13 10:05:47 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 10:05:47 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 10:05:47 @./script/server/login.lua:67] login listen client success
[E:runtime][2018-04-13 10:05:48 @./script/server/agent.lua:59] agent start success
[E:runtime][2018-04-13 10:06:18 @user.server.startup:49] server/scene connect server:master ipc://master.ipc failed:Connection refused
[E:runtime][2018-04-13 10:06:18 @./script/server/login.lua:68] login listen client success
[E:runtime][2018-04-13 10:06:18 @./script/server/agent.lua:59] agent start success
[E:runtime][2018-04-13 10:07:04 @./script/server/login.lua:72] login listen client success
[E:runtime][2018-04-13 10:07:04 @user.server.startup:49] server/agent connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 10:07:05 @./script/server/agent.lua:59] agent start success
[E:runtime][2018-04-13 10:08:13 @./script/server/login.lua:68] login listen client success
[E:runtime][2018-04-13 10:08:14 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 10:08:14 @./script/server/agent.lua:59] agent start success
[E:runtime][2018-04-13 10:08:15 @./script/server/login.lua:41] user.handler.login_handler:13: attempt to index a nil value (global 'event')
stack traceback:
	user.handler.login_handler:13: in function <user.handler.login_handler:10>
	[C]: in function 'xpcall'
	./script/server/login.lua:39: in function <./script/server/login.lua:38>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 10:09:27 @./script/server/login.lua:68] login listen client success
[E:runtime][2018-04-13 10:09:27 @./script/server/agent.lua:59] agent start success
[E:runtime][2018-04-13 10:09:31 @user.handler.login_handler:14] cid:4999 addr:127.0.0.1:47198 enter
[E:runtime][2018-04-13 11:34:29 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 11:34:29 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 11:37:21 @./script/server/login.lua:69] login listen client success
[E:runtime][2018-04-13 11:37:21 @user.server.startup:49] server/agent connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 11:37:21 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 11:37:21 @user.server.startup:49] server/agent connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 11:37:22 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:37:22 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:37:30 @user.handler.login_handler:19] cid:4999 addr:127.0.0.1:47460 enter
[E:runtime][2018-04-13 11:38:03 @./script/server/login.lua:70] login listen client success
[E:runtime][2018-04-13 11:38:03 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:38:03 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:38:05 @user.handler.login_handler:19] cid:4999 addr:127.0.0.1:47472 enter
[E:runtime][2018-04-13 11:38:15 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 11:38:15 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 11:38:15 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 11:38:15 @./script/server/login.lua:70] login listen client success
[E:runtime][2018-04-13 11:38:16 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:38:16 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:38:18 @user.handler.login_handler:19] cid:4999 addr:127.0.0.1:47486 enter
[E:runtime][2018-04-13 11:38:48 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 11:38:48 @./script/server/login.lua:70] login listen client success
[E:runtime][2018-04-13 11:38:48 @user.server.startup:49] server/agent connect server:master ipc://master.ipc failed:Connection refused
[E:runtime][2018-04-13 11:38:49 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:38:49 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:38:50 @user.handler.login_handler:19] cid:4999 addr:127.0.0.1:47498 enter
[E:runtime][2018-04-13 11:39:10 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 11:39:10 @./script/server/login.lua:69] login listen client success
[E:runtime][2018-04-13 11:39:10 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:39:11 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:39:13 @user.handler.login_handler:19] cid:4999 addr:127.0.0.1:47510 enter
[E:runtime][2018-04-13 11:39:14 @./script/server/login.lua:33] ./lualib/route.lua:21: attempt to index a nil value (global 'protocol')
stack traceback:
	./lualib/route.lua:21: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:39:14 @./script/server/login.lua:33] ./lualib/route.lua:21: attempt to index a nil value (global 'protocol')
stack traceback:
	./lualib/route.lua:21: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:39:30 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 11:39:30 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 11:39:30 @./script/server/login.lua:69] login listen client success
[E:runtime][2018-04-13 11:39:30 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:39:31 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:39:33 @user.handler.login_handler:19] cid:4999 addr:127.0.0.1:47524 enter
[E:runtime][2018-04-13 11:39:34 @./script/server/login.lua:33] ./lualib/route.lua:22: attempt to call a nil value (field '?')
stack traceback:
	./lualib/route.lua:22: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:39:34 @./script/server/login.lua:33] ./lualib/route.lua:22: attempt to call a nil value (field '?')
stack traceback:
	./lualib/route.lua:22: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:40:11 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 11:40:11 @user.server.startup:49] server/agent connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 11:40:11 @./script/server/login.lua:69] login listen client success
[E:runtime][2018-04-13 11:40:11 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 11:40:12 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:40:12 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:40:14 @user.handler.login_handler:19] cid:4999 addr:127.0.0.1:47538 enter
[E:runtime][2018-04-13 11:40:17 @./script/server/login.lua:33] ./lualib/route.lua:22: attempt to call a nil value (field '?')
stack traceback:
	./lualib/route.lua:22: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:40:17 @./script/server/login.lua:33] ./lualib/route.lua:22: attempt to call a nil value (field '?')
stack traceback:
	./lualib/route.lua:22: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:40:37 @./script/server/login.lua:69] login listen client success
[E:runtime][2018-04-13 11:40:37 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 11:40:38 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:40:38 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:40:39 @user.handler.login_handler:19] cid:4999 addr:127.0.0.1:47550 enter
[E:runtime][2018-04-13 11:40:41 @./script/server/login.lua:33] ./lualib/route.lua:23: attempt to call a nil value (field '?')
stack traceback:
	./lualib/route.lua:23: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:40:41 @./script/server/login.lua:33] ./lualib/route.lua:23: attempt to call a nil value (field '?')
stack traceback:
	./lualib/route.lua:23: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:40:49 @./script/server/login.lua:69] login listen client success
[E:runtime][2018-04-13 11:40:49 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:40:50 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:40:52 @user.handler.login_handler:19] cid:4999 addr:127.0.0.1:47562 enter
[E:runtime][2018-04-13 11:40:53 @./script/server/login.lua:33] ./lualib/route.lua:23: attempt to call a nil value (field '?')
stack traceback:
	./lualib/route.lua:23: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:40:53 @./script/server/login.lua:33] ./lualib/route.lua:23: attempt to call a nil value (field '?')
stack traceback:
	./lualib/route.lua:23: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:41:26 @./script/server/login.lua:69] login listen client success
[E:runtime][2018-04-13 11:41:26 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 11:41:26 @user.server.startup:49] server/agent connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 11:41:27 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:41:27 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:41:29 @user.handler.login_handler:19] cid:4999 addr:127.0.0.1:47576 enter
[E:runtime][2018-04-13 11:41:30 @./script/server/login.lua:33] ./lualib/protocol.lua:104: bad argument #1 to 'decode' (number expected, got light userdata)
stack traceback:
	[C]: in method 'decode'
	./lualib/protocol.lua:104: in field '?'
	./lualib/route.lua:23: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:41:30 @./script/server/login.lua:33] ./lualib/protocol.lua:104: bad argument #1 to 'decode' (number expected, got light userdata)
stack traceback:
	[C]: in method 'decode'
	./lualib/protocol.lua:104: in field '?'
	./lualib/route.lua:23: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:42:42 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 11:42:43 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:No such file or directory
[E:runtime][2018-04-13 11:42:43 @./script/server/login.lua:69] login listen client success
[E:runtime][2018-04-13 11:42:43 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:42:44 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:42:44 @user.handler.login_handler:19] cid:4999 addr:127.0.0.1:47590 enter
[E:runtime][2018-04-13 11:42:46 @./script/server/login.lua:33] ./lualib/route.lua:24: attempt to call a nil value (field '?')
stack traceback:
	./lualib/route.lua:24: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:42:46 @./script/server/login.lua:33] ./lualib/route.lua:24: attempt to call a nil value (field '?')
stack traceback:
	./lualib/route.lua:24: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:43:03 @./script/server/login.lua:69] login listen client success
[E:runtime][2018-04-13 11:43:03 @user.server.startup:49] server/agent connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 11:43:03 @user.server.startup:49] server/agent connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 11:43:04 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:43:04 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:43:05 @user.handler.login_handler:19] cid:4999 addr:127.0.0.1:47602 enter
[E:runtime][2018-04-13 11:43:07 @./script/server/login.lua:33] ./lualib/route.lua:22: attempt to call a nil value (field '?')
stack traceback:
	./lualib/route.lua:22: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:43:07 @./script/server/login.lua:33] ./lualib/route.lua:22: attempt to call a nil value (field '?')
stack traceback:
	./lualib/route.lua:22: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:43:15 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 11:43:15 @./script/server/login.lua:69] login listen client success
[E:runtime][2018-04-13 11:43:15 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:43:16 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:43:17 @user.handler.login_handler:19] cid:4999 addr:127.0.0.1:47616 enter
[E:runtime][2018-04-13 11:43:18 @./script/server/login.lua:33] ./lualib/route.lua:23: attempt to call a nil value (field '?')
stack traceback:
	./lualib/route.lua:23: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:43:18 @./script/server/login.lua:33] ./lualib/route.lua:23: attempt to call a nil value (field '?')
stack traceback:
	./lualib/route.lua:23: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:44:00 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 11:44:00 @./script/server/login.lua:69] login listen client success
[E:runtime][2018-04-13 11:44:00 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 11:44:01 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:44:01 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:44:03 @user.handler.login_handler:19] cid:4999 addr:127.0.0.1:47628 enter
[E:runtime][2018-04-13 11:44:04 @./script/server/login.lua:33] ./lualib/route.lua:23: attempt to index a nil value (global 'event')
stack traceback:
	./lualib/route.lua:23: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:44:04 @./script/server/login.lua:33] ./lualib/route.lua:23: attempt to index a nil value (global 'event')
stack traceback:
	./lualib/route.lua:23: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:44:15 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 11:44:16 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 11:44:16 @./script/server/login.lua:69] login listen client success
[E:runtime][2018-04-13 11:44:16 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:44:17 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:44:17 @user.handler.login_handler:19] cid:4999 addr:127.0.0.1:47642 enter
[E:runtime][2018-04-13 11:44:18 @./script/server/login.lua:33] ./lualib/route.lua:24: bad argument #3 to 'format' (number expected, got string)
stack traceback:
	[C]: in function 'string.format'
	./lualib/route.lua:24: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:44:18 @./script/server/login.lua:33] ./lualib/route.lua:24: bad argument #3 to 'format' (number expected, got string)
stack traceback:
	[C]: in function 'string.format'
	./lualib/route.lua:24: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:44:33 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 11:44:33 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 11:44:33 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 11:44:33 @./script/server/login.lua:69] login listen client success
[E:runtime][2018-04-13 11:44:34 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:44:34 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:44:35 @user.handler.login_handler:19] cid:4999 addr:127.0.0.1:47654 enter
[E:runtime][2018-04-13 11:44:36 @./lualib/route.lua:24] no such id:1 proto:c2s_auth 
[E:runtime][2018-04-13 11:44:36 @./lualib/route.lua:24] no such id:1 proto:c2s_auth 
[E:runtime][2018-04-13 11:46:41 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 11:46:41 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 11:46:41 @./script/server/login.lua:69] login listen client success
[E:runtime][2018-04-13 11:46:42 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:46:42 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:47:04 @user.handler.login_handler:19] cid:4999 addr:127.0.0.1:47670 enter
[E:runtime][2018-04-13 11:47:05 @./lualib/route.lua:24] no such id:1 proto:c2s_create_role 
[E:runtime][2018-04-13 11:47:05 @./lualib/route.lua:24] no such id:1 proto:c2s_create_role 
[E:runtime][2018-04-13 11:48:45 @user.handler.login_handler:23] cid:4999 enter
[E:runtime][2018-04-13 11:48:46 @user.handler.login_handler:19] cid:4998 addr:127.0.0.1:47676 enter
[E:runtime][2018-04-13 11:48:47 @./script/server/login.lua:33] user.handler.login_handler:48: attempt to call a nil value (field 'fetch_login_user_with_account')
stack traceback:
	user.handler.login_handler:48: in field '?'
	./lualib/route.lua:27: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:48:47 @./script/server/login.lua:33] user.handler.login_handler:48: attempt to call a nil value (field 'fetch_login_user_with_account')
stack traceback:
	user.handler.login_handler:48: in field '?'
	./lualib/route.lua:27: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:50:17 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 11:50:17 @user.server.startup:49] server/scene connect server:master ipc://master.ipc failed:Connection refused
[E:runtime][2018-04-13 11:50:17 @./script/server/login.lua:69] login listen client success
[E:runtime][2018-04-13 11:50:17 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:50:18 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:50:20 @user.handler.login_handler:19] cid:4999 addr:127.0.0.1:47692 enter
[E:runtime][2018-04-13 11:50:24 @./script/server/login.lua:33] ./lualib/model.lua:33: attempt to index a nil value (local 'same_k')
stack traceback:
	./lualib/model.lua:33: in function 'model.fetch_login_user_with_account'
	user.handler.login_handler:48: in field '?'
	./lualib/route.lua:27: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:50:24 @./script/server/login.lua:33] ./lualib/model.lua:33: attempt to index a nil value (local 'same_k')
stack traceback:
	./lualib/model.lua:33: in function 'model.fetch_login_user_with_account'
	user.handler.login_handler:48: in field '?'
	./lualib/route.lua:27: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:51:59 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 11:51:59 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 11:51:59 @./script/server/login.lua:69] login listen client success
[E:runtime][2018-04-13 11:52:00 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:52:00 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:52:01 @user.handler.login_handler:19] cid:4999 addr:127.0.0.1:47706 enter
[E:runtime][2018-04-13 11:52:03 @./script/server/login.lua:33] user.handler.login_handler:72: attempt to index a nil value (local 'login_user')
stack traceback:
	user.handler.login_handler:72: in field '?'
	./lualib/route.lua:27: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:52:03 @./script/server/login.lua:33] user.handler.login_handler:72: attempt to index a nil value (local 'login_user')
stack traceback:
	user.handler.login_handler:72: in field '?'
	./lualib/route.lua:27: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:53:21 @./script/server/login.lua:69] login listen client success
[E:runtime][2018-04-13 11:53:21 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 11:53:21 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 11:53:21 @./lualib/event.lua:71] user.object:82: attempt to index a nil value (global 'model')
stack traceback:
	user.object:82: in method 'inherit'
	user.module.login_user:9: in local 'func'
	./lualib/import.lua:20: in upvalue 'loadfile'
	./lualib/import.lua:34: in function 'import.import'
	user.handler.login_handler:7: in local 'func'
	./lualib/import.lua:20: in upvalue 'loadfile'
	./lualib/import.lua:34: in function 'import.import'
	./script/server/login.lua:72: in upvalue 'func'
	./lualib/event.lua:49: in function <./lualib/event.lua:48>
[E:runtime][2018-04-13 11:53:21 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:53:22 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:54:17 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 11:54:18 @user.server.startup:49] server/agent connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 11:54:18 @./script/server/login.lua:69] login listen client success
[E:runtime][2018-04-13 11:54:18 @./lualib/event.lua:71] ./lualib/model.lua:13: name:login_user already register
stack traceback:
	[C]: in function 'assert'
	./lualib/model.lua:13: in function 'model.register_binder'
	user.object:82: in method 'inherit'
	user.module.login_user:9: in local 'func'
	./lualib/import.lua:20: in upvalue 'loadfile'
	./lualib/import.lua:34: in function 'import'
	user.handler.login_handler:7: in local 'func'
	./lualib/import.lua:20: in upvalue 'loadfile'
	./lualib/import.lua:34: in function 'import'
	./script/server/login.lua:72: in upvalue 'func'
	./lualib/event.lua:49: in function <./lualib/event.lua:48>
[E:runtime][2018-04-13 11:54:18 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:54:19 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:54:34 @user.server.startup:49] server/scene connect server:master ipc://master.ipc failed:Connection refused
[E:runtime][2018-04-13 11:54:34 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 11:54:34 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 11:54:34 @./script/server/login.lua:69] login listen client success
[E:runtime][2018-04-13 11:54:35 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:54:35 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:54:38 @user.handler.login_handler:20] cid:4999 addr:127.0.0.1:47744 enter
[E:runtime][2018-04-13 11:54:39 @./script/server/login.lua:33] user.module.login_user:40: attempt to call a nil value (field 'get_mongodb')
stack traceback:
	user.module.login_user:40: in method 'auth'
	user.handler.login_handler:74: in field '?'
	./lualib/route.lua:27: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:54:39 @./script/server/login.lua:33] user.module.login_user:40: attempt to call a nil value (field 'get_mongodb')
stack traceback:
	user.module.login_user:40: in method 'auth'
	user.handler.login_handler:74: in field '?'
	./lualib/route.lua:27: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:54:57 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 11:54:57 @user.server.startup:49] server/scene connect server:master ipc://master.ipc failed:Connection refused
[E:runtime][2018-04-13 11:54:57 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:No such file or directory
[E:runtime][2018-04-13 11:54:57 @./script/server/login.lua:69] login listen client success
[E:runtime][2018-04-13 11:54:58 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:54:58 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:55:01 @user.handler.login_handler:20] cid:4999 addr:127.0.0.1:47756 enter
[E:runtime][2018-04-13 11:55:02 @./script/server/login.lua:33] ./lualib/event.lua:240: cannot wait in main co,wait op should run in fork
stack traceback:
	[C]: in function 'error'
	./lualib/event.lua:240: in function 'event.wait'
	./lualib/mongo.lua:97: in upvalue 'find'
	./lualib/mongo.lua:109: in function 'mongo.findOne'
	user.database_object:16: in method 'load'
	user.module.login_user:41: in method 'auth'
	user.handler.login_handler:74: in field '?'
	./lualib/route.lua:27: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:55:02 @./script/server/login.lua:33] ./lualib/event.lua:240: cannot wait in main co,wait op should run in fork
stack traceback:
	[C]: in function 'error'
	./lualib/event.lua:240: in function 'event.wait'
	./lualib/mongo.lua:97: in upvalue 'find'
	./lualib/mongo.lua:109: in function 'mongo.findOne'
	user.database_object:16: in method 'load'
	user.module.login_user:41: in method 'auth'
	user.handler.login_handler:74: in field '?'
	./lualib/route.lua:27: in function 'route.dispatch_client'
	[C]: in function 'xpcall'
	./script/server/login.lua:31: in function <./script/server/login.lua:30>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:55:02 @./lualib/event.lua:349] ./lualib/channel.lua:86: luaclib/lua-dump.c:965 unpack error at:0 : expect {,unknown char:<\0> 
stack traceback:
	[C]: in function 'dump.core.unpack'
	./lualib/channel.lua:86: in function 'channel.data'
	./lualib/event.lua:333: in function <./lualib/event.lua:331>
	[C]: in function 'xpcall'
	./lualib/event.lua:347: in function <./lualib/event.lua:341>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:55:02 @./lualib/event.lua:349] ./lualib/channel.lua:86: luaclib/lua-dump.c:965 unpack error at:0 : expect {,unknown char:<\1> 
stack traceback:
	[C]: in function 'dump.core.unpack'
	./lualib/channel.lua:86: in function 'channel.data'
	./lualib/event.lua:333: in function <./lualib/event.lua:331>
	[C]: in function 'xpcall'
	./lualib/event.lua:347: in function <./lualib/event.lua:341>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:56:13 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 11:56:13 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 11:56:13 @./script/server/login.lua:68] login listen client success
[E:runtime][2018-04-13 11:56:14 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:56:14 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:56:16 @user.handler.login_handler:20] cid:4999 addr:127.0.0.1:47772 enter
[E:runtime][2018-04-13 11:56:17 @./lualib/event.lua:349] ./lualib/channel.lua:86: luaclib/lua-dump.c:965 unpack error at:0 : expect {,unknown char:<\2> 
stack traceback:
	[C]: in function 'table.decode'
	./lualib/channel.lua:86: in function 'channel.data'
	./lualib/event.lua:333: in function <./lualib/event.lua:331>
	[C]: in function 'xpcall'
	./lualib/event.lua:347: in function <./lualib/event.lua:341>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:56:17 @./lualib/event.lua:349] ./lualib/channel.lua:86: luaclib/lua-dump.c:965 unpack error at:0 : expect {,unknown char:<\3> 
stack traceback:
	[C]: in function 'table.decode'
	./lualib/channel.lua:86: in function 'channel.data'
	./lualib/event.lua:333: in function <./lualib/event.lua:331>
	[C]: in function 'xpcall'
	./lualib/event.lua:347: in function <./lualib/event.lua:341>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:58:00 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 11:58:00 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 11:58:00 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 11:58:00 @./script/server/login.lua:68] login listen client success
[E:runtime][2018-04-13 11:58:01 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:58:01 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 11:58:03 @user.handler.login_handler:20] cid:4999 addr:127.0.0.1:47786 enter
[E:runtime][2018-04-13 11:58:05 @./lualib/event.lua:349] ./lualib/channel.lua:86: luaclib/lua-dump.c:965 unpack error at:0 : expect {,unknown char:<\4> 
stack traceback:
	[C]: in function 'dump.core.unpack'
	./lualib/channel.lua:86: in function 'channel.data'
	./lualib/event.lua:333: in function <./lualib/event.lua:331>
	[C]: in function 'xpcall'
	./lualib/event.lua:347: in function <./lualib/event.lua:341>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 11:58:05 @./lualib/event.lua:349] ./lualib/channel.lua:86: luaclib/lua-dump.c:965 unpack error at:0 : expect {,unknown char:<\5> 
stack traceback:
	[C]: in function 'dump.core.unpack'
	./lualib/channel.lua:86: in function 'channel.data'
	./lualib/event.lua:333: in function <./lualib/event.lua:331>
	[C]: in function 'xpcall'
	./lualib/event.lua:347: in function <./lualib/event.lua:341>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 12:00:49 @./script/server/login.lua:68] login listen client success
[E:runtime][2018-04-13 12:00:49 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 12:00:49 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 12:00:52 @user.handler.login_handler:20] cid:4999 addr:127.0.0.1:47804 enter
[E:runtime][2018-04-13 12:00:53 @./lualib/event.lua:349] ./lualib/channel.lua:86: luaclib/lua-dump.c:965 unpack error at:0 : expect {,unknown char:<\6> 
stack traceback:
	[C]: in function 'dump.core.unpack'
	./lualib/channel.lua:86: in function 'channel.data'
	./lualib/event.lua:333: in function <./lualib/event.lua:331>
	[C]: in function 'xpcall'
	./lualib/event.lua:347: in function <./lualib/event.lua:341>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 12:00:53 @./lualib/event.lua:349] ./lualib/channel.lua:86: luaclib/lua-dump.c:965 unpack error at:0 : expect {,unknown char:<\7> 
stack traceback:
	[C]: in function 'dump.core.unpack'
	./lualib/channel.lua:86: in function 'channel.data'
	./lualib/event.lua:333: in function <./lualib/event.lua:331>
	[C]: in function 'xpcall'
	./lualib/event.lua:347: in function <./lualib/event.lua:341>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 12:02:46 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 12:02:46 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 12:02:46 @./script/server/login.lua:68] login listen client success
[E:runtime][2018-04-13 12:02:47 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 12:02:47 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 12:02:49 @user.handler.login_handler:20] cid:4999 addr:127.0.0.1:47820 enter
[E:runtime][2018-04-13 12:03:23 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 12:03:23 @./script/server/login.lua:68] login listen client success
[E:runtime][2018-04-13 12:03:23 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 12:03:23 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 12:03:24 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 12:03:24 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 12:03:25 @user.handler.login_handler:20] cid:4999 addr:127.0.0.1:47834 enter
[E:runtime][2018-04-13 12:04:05 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 12:04:05 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 12:04:05 @./script/server/login.lua:68] login listen client success
[E:runtime][2018-04-13 12:04:05 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 12:04:06 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 12:04:06 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 12:04:07 @user.handler.login_handler:20] cid:4999 addr:127.0.0.1:47846 enter
[E:runtime][2018-04-13 12:04:33 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 12:04:33 @./script/server/login.lua:68] login listen client success
[E:runtime][2018-04-13 12:04:33 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 12:04:33 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 12:04:35 @user.handler.login_handler:20] cid:4999 addr:127.0.0.1:47860 enter
[E:runtime][2018-04-13 12:05:37 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 12:05:37 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 12:05:37 @./script/server/login.lua:68] login listen client success
[E:runtime][2018-04-13 12:05:38 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 12:05:40 @user.handler.login_handler:20] cid:4999 addr:127.0.0.1:47874 enter
[E:runtime][2018-04-13 12:05:42 @./lualib/event.lua:349] user.module.login_user:30: attempt to call a nil value (field 'get_mongod')
stack traceback:
	user.module.login_user:30: in local 'info'
	./lualib/event.lua:317: in function <./lualib/event.lua:310>
	[C]: in function 'xpcall'
	./lualib/event.lua:347: in function <./lualib/event.lua:341>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 12:05:43 @./lualib/event.lua:349] user.module.login_user:30: attempt to call a nil value (field 'get_mongod')
stack traceback:
	user.module.login_user:30: in local 'info'
	./lualib/event.lua:317: in function <./lualib/event.lua:310>
	[C]: in function 'xpcall'
	./lualib/event.lua:347: in function <./lualib/event.lua:341>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 12:05:44 @./lualib/event.lua:349] user.module.login_user:30: attempt to call a nil value (field 'get_mongod')
stack traceback:
	user.module.login_user:30: in local 'info'
	./lualib/event.lua:317: in function <./lualib/event.lua:310>
	[C]: in function 'xpcall'
	./lualib/event.lua:347: in function <./lualib/event.lua:341>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 12:05:45 @./lualib/event.lua:349] user.module.login_user:30: attempt to call a nil value (field 'get_mongod')
stack traceback:
	user.module.login_user:30: in local 'info'
	./lualib/event.lua:317: in function <./lualib/event.lua:310>
	[C]: in function 'xpcall'
	./lualib/event.lua:347: in function <./lualib/event.lua:341>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 12:05:46 @./lualib/event.lua:349] user.module.login_user:30: attempt to call a nil value (field 'get_mongod')
stack traceback:
	user.module.login_user:30: in local 'info'
	./lualib/event.lua:317: in function <./lualib/event.lua:310>
	[C]: in function 'xpcall'
	./lualib/event.lua:347: in function <./lualib/event.lua:341>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 12:05:47 @./lualib/event.lua:349] user.module.login_user:30: attempt to call a nil value (field 'get_mongod')
stack traceback:
	user.module.login_user:30: in local 'info'
	./lualib/event.lua:317: in function <./lualib/event.lua:310>
	[C]: in function 'xpcall'
	./lualib/event.lua:347: in function <./lualib/event.lua:341>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 12:05:48 @./lualib/event.lua:349] user.module.login_user:30: attempt to call a nil value (field 'get_mongod')
stack traceback:
	user.module.login_user:30: in local 'info'
	./lualib/event.lua:317: in function <./lualib/event.lua:310>
	[C]: in function 'xpcall'
	./lualib/event.lua:347: in function <./lualib/event.lua:341>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 12:05:49 @./lualib/event.lua:349] user.module.login_user:30: attempt to call a nil value (field 'get_mongod')
stack traceback:
	user.module.login_user:30: in local 'info'
	./lualib/event.lua:317: in function <./lualib/event.lua:310>
	[C]: in function 'xpcall'
	./lualib/event.lua:347: in function <./lualib/event.lua:341>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 12:05:50 @./lualib/event.lua:349] user.module.login_user:30: attempt to call a nil value (field 'get_mongod')
stack traceback:
	user.module.login_user:30: in local 'info'
	./lualib/event.lua:317: in function <./lualib/event.lua:310>
	[C]: in function 'xpcall'
	./lualib/event.lua:347: in function <./lualib/event.lua:341>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 12:05:51 @./lualib/event.lua:349] user.module.login_user:30: attempt to call a nil value (field 'get_mongod')
stack traceback:
	user.module.login_user:30: in local 'info'
	./lualib/event.lua:317: in function <./lualib/event.lua:310>
	[C]: in function 'xpcall'
	./lualib/event.lua:347: in function <./lualib/event.lua:341>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 12:05:52 @./lualib/event.lua:349] user.module.login_user:30: attempt to call a nil value (field 'get_mongod')
stack traceback:
	user.module.login_user:30: in local 'info'
	./lualib/event.lua:317: in function <./lualib/event.lua:310>
	[C]: in function 'xpcall'
	./lualib/event.lua:347: in function <./lualib/event.lua:341>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 12:05:53 @./lualib/event.lua:349] user.module.login_user:30: attempt to call a nil value (field 'get_mongod')
stack traceback:
	user.module.login_user:30: in local 'info'
	./lualib/event.lua:317: in function <./lualib/event.lua:310>
	[C]: in function 'xpcall'
	./lualib/event.lua:347: in function <./lualib/event.lua:341>
	[C]: in method 'dispatch'
	./lualib/event.lua:263: in function 'event.dispatch'
	lualib/bootstrap.lua:106: in main chunk
[E:runtime][2018-04-13 12:05:54 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 12:05:54 @user.server.startup:49] server/agent connect server:login ipc://login.ipc failed:Connection refused
[E:runtime][2018-04-13 12:05:54 @./script/server/login.lua:68] login listen client success
[E:runtime][2018-04-13 12:05:55 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 12:05:55 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 12:05:57 @user.handler.login_handler:20] cid:4999 addr:127.0.0.1:47886 enter
[E:runtime][2018-04-13 12:06:17 @user.handler.login_handler:24] cid:4999 enter
[E:runtime][2018-04-13 12:07:25 @./script/server/login.lua:68] login listen client success
[E:runtime][2018-04-13 12:07:25 @user.server.startup:49] server/scene connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 12:07:25 @user.server.startup:49] server/agent connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 12:07:25 @user.server.startup:49] server/agent connect server:world ipc://world.ipc failed:Connection refused
[E:runtime][2018-04-13 12:07:26 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 12:07:26 @./script/server/agent.lua:58] agent start success
[E:runtime][2018-04-13 12:07:28 @user.handler.login_handler:20] cid:4999 addr:127.0.0.1:47902 enter
